massa-metrics/src/lib.rs

```diff
@@ -82,14 +98,30 @@ pub struct MassaMetrics {
 
     final_cursor_thread: IntGauge,
     final_cursor_period: IntGauge,
+
+    // peer bandwidth (bytes sent, bytes received)
+    // todo remove Arc<RwLock<...>> ?
+    peers_bandwidth: Arc<RwLock<HashMap<String, (IntCounter, IntCounter)>>>,
```

Comment:
Actually i can remove this Arc<RwLock<>> because only the connectivity thread will edit this HashMap